ci-operator/step-registry/cucushift/installer/check/azure/osimage/cucushift-installer-check-azure-osimage-commands.sh

```diff
@@ -98,6 +98,7 @@ worker_generation=$(az vm image show --urn ${worker_image_urn} --query hyperVGen
 critical_check_result=0
 
 echo "---------- Check worker nodes urn and hyperV generation ----------"
+echo "Expected worker_image_urn: ${worker_image_urn}, expected worker_generation: ${worker_generation}"
```

Comment:
Can we apply this step - `cucushift-installer-check-azure-osimage` to `azure-ovn-marketplace-images` pre-submit ci jobs ?